packages/es-components/src/components/containers/popover/Popover.js

```diff
@@ -189,7 +186,7 @@ function Popover(props) {
           popperRef.current = elem;
         }}
       >
-        <PopoverContainer role="dialog" ref={contentRef}>
+        <div role="dialog" ref={contentRef}>
```

Comment:
Yeah, I'm not loving the div either, but its unfortunately a necessary thing. The ref does work on other elements, I put it in that div so that it would find any button or link in either the header or the body. I can move it, but if I do I think it makes the most sense for it to be on the `PopoverBody` element.